### PR TITLE
Fixes to user migration doc   doc 6860

### PR DESCRIPTION
**Pull request by zuluecho9**
Jira DOC 6860. Let's just do PR review for now and I will be getting a bigger peer edit later. Fixes include: 

Adding more explanatory info towards start. 
Adding details/tips for each page of the UI wizard, and screenshots.

### CODE REVIEW

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 1]
```diff
@@ -2,20 +2,15 @@
...
-  user
-  model
 ---
 
 Starting April 12, 2021, we're allowing some customers who have users on our original user model to self-serve and migrate those users to be on the New Relic One user model.
 
 ## Background [#background]
 
-On July 30 2020, we released a new and improved account/user model called the New Relic One user model. This newer model offers a simpler, more efficient way to manage users and what they have access to.
+On July 30 2020, we released a new and improved account/user model called the New Relic One user model. This newer model offers a simpler, more efficient way to manage users and their access to roles and accounts.
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:11]: Shouldn't it be "On July 30, 2020,"?

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 2]
```diff
@@ -39,13 +34,13 @@ Factors affecting access to the migration wizard:
 
 ## Before you start [#before-beginning]
 
-At a certain point in the user migration procedure, you'll have a choice to either grant all your users access to all accounts in your organization, or to customize which accounts your users have access to. 
+During the user migration procedure, you'll have a choice to either grant all your users complete access to all your organization's accounts, or to customize which roles and accounts your users have access to (aka, assign **access grants**).
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:37]: Not a fan of aka

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 3]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
-If your email address is associated with multiple logins, you can verify your email to see those accounts. In the example above, the “Original New Relic account” refers to that user record being on the older, original account/user model. 
+If a user with access to multiple organizations is not migrated for all organizations, they'll retain an "original" user record, in addition to the new record created. When logging in, they'll have a choice of which to log into. 
     </figcaption>
+
+
+## Step 6: Access settings [#access-settings]
+
+![User migration page 6](./images/user-migration-page-6.png "user-migration-page-6.png")
+
+This step allows you to determine the level of access their users have. If you need to map user access to specific accounts and roles, then you'll want to set up groups and access grants at this stage.
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:144]: their users --> your users

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 4]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
     </figcaption>
+
+
+## Step 6: Access settings [#access-settings]
+
+![User migration page 6](./images/user-migration-page-6.png "user-migration-page-6.png")
+
+This step allows you to determine the level of access their users have. If you need to map user access to specific accounts and roles, then you'll want to set up groups and access grants at this stage. 
+
+You'll need to create an access grant for each account that they want a group to have access to. For example, a DevOps group might have five different access grants, each giving the group access to a different account.
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:146]: "for each account that *you* want a group to have access to"?

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 5]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
+
+## Step 6: Access settings [#access-settings]
+
+![User migration page 6](./images/user-migration-page-6.png "user-migration-page-6.png")
+
+This step allows you to determine the level of access their users have. If you need to map user access to specific accounts and roles, then you'll want to set up groups and access grants at this stage. 
+
+You'll need to create an access grant for each account that they want a group to have access to. For example, a DevOps group might have five different access grants, each giving the group access to a different account.  
+
+This is a procedure done separately from the user migration wizard procedure. For more on how to create access grants, see the [user management tutorial](/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model/#group-access).
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:148]: Maybe we can reword this to avoid the passive voice here...

Something like: "This procedure is done separately from the..."

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 6]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
 
-Once your users are migrated to the new model, you can find and manage them by clicking the account dropdown, clicking **Administration**, and using these UI pages: 
+If you've successfully completed migration, learn [how to manage your users](#manage-users). 
 
-* **User management**: use this to view and add users, change their type (basic vs. full), change their group, and approve user upgrade requests. 
-* **Organization and access**: use this to create access grants (granting groups access to roles and accounts), and configure authentication domains (SAML SSO settings and SCIM settings, and more). 
+## Step 1: Create admins [#page1]
 
-For more about these tools, see the [new user management docs](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/).
+![User migration page 1](./images/user-migration-page-1.png "user-migration-page-1.png")
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:68]: This image is missing a caption.

Comment (zuluecho9) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:68]: i was not using captions for most of them, as it felt pretty obvious I was just showing images from the walkthrough.

Comment (paperclypse) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:68]: Also, I think this image might be easier to see if you crop out the Migration FAQ from the right side of the image.

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 7]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
+	 ![A screenshot of what is shown when you have an email address associated with multiple New Relic logins](./images/login-multiple-accounts-view.png "Login UI for email addresses with multiple user records")
+    <figcaption>
+	  If a user on the new model has been created and the migration process hasn't been completed, they may have access to the original user record and the new login(s). 
+    </figcaption>
+* If you plan on migrating only a portion of your users to the new user model to start, we recommend leaving some original model admins so that you have an admin to manage the original model users. 
+
+
+## Step 2: Set up organization [#page2]
+
+![User migration page 2](./images/user-migration-page-2.png "user-migration-page-2.png")
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:82]: I think you can crop out the left side of this image

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 8]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
+
+You may choose a) a guided setup that allows more configuration options, or b) an automatic setup with fewer steps. Some notes on the automatic setup:
+
+* It won't transition SSO or [AUM-implemented SCIM configurations](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign). Instead, users will be provisioned manually and will log in manually with their username and password. 
+* It provides all users with access to all accounts. If you use SSO or AUM today or require some accounts to be inaccessible by certain user groups, don't use the automatic option.
+
+
+## Step 3: Name your organization [#page3]
+
+![User migration page 3](./images/user-migration-page-3.png "user-migration-page-3.png")
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:92]: I think you can crop out the left nav from this image

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 9]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
+
+If you chose **Manually** for [managing users](#managing-users), you have a choice for how those users log in. You can select either a) email/password login or b) single sign on (SSO). Note that SSO is available only for organizations with Pro or Enterprise edition.
+
+You can select multiple authentication methods based on your needs. For example, you might use SSO for your employees and username/password for contractors. 
+
+If you select SSO, you must complete [additional configuration steps](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#saml). You can complete the procedure for setting up SSO, and then come back to the migration wizard to continue the migration process. If you select more than one authentication method, note that you’ll need to add a new [authentication domain](/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more/#auth-domain-definition).
+
+## Step 5: Import existing users
+
+![User migration page 5](./images/user-migration-page-5.png "user-migration-page-5.png")
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:120]: I think you can crop ou the left nav from this image. Also there's no caption on this one.

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 10]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
+
+## Step 3: Name your organization [#page3]
+
+![User migration page 3](./images/user-migration-page-3.png "user-migration-page-3.png")
+
+Name your organization something descriptive and easily recognizable.
+
+## Step 4: Authentication domain settings [#page4]
+
+![User migration page 4](./images/user-migration-page-4.png "user-migration-page-4.png")
```
[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:98]: Also, no caption. If this is a deliberate choice, that's fine.

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx (hunk 8 above)
Comment (paperclypse) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:92]: Another no caption

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 11]
```diff
@@ -58,33 +53,127 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
+
+## Step 7: Migrate user assets [#migrate-assets]
+
+![User migration page 7](./images/user-migration-page-7.png "user-migration-page-7.png")
+
+Upon completion of this step, personal assets, such as dashboards and favorites, are migrated to new user records and the original user record is deleted.
+
+If a user has access to several organizations (for example, if that user is a contractor), the user won't be fully deleted until all organizations migrate their users. It's possible that such a user will find themselves with both the original user record and one or more new user records (see image below). 
+
+ ![A screenshot of what is shown when you have an email address associated with multiple New Relic logins](./images/login-multiple-accounts-view.png "Login UI for email addresses with multiple user records")
```
Comment (paperclypse) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:158]: I don't think we need this image twice. I think it's fine to just have it the first time toward the start of this doc

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx (hunk 9 above)
Comment (paperclypse) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:120]: This comment applies to all of the other similar images too.

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx (hunk 8 above)
Comment (zuluecho9) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:92]: i think it's possible to do so, but I think it's helpful in letting them know 'this is part of the user migration wizard in UI'. I don't think it detracts. (I think actually the more long horizontal-wise rectangles display a bit better on page but that's a separate issue).

**Review by paperclypse — APPROVED** [review]
This is a big improvement over what I saw before. I'm approving it. You can take or leave my comments, as you like.

**Review by paperclypse — APPROVED** [review]

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx [hunk 12]
```diff
@@ -58,33 +53,114 @@ If you'll need to do a good amount of partitioning of users' access to accounts
...
+
+If you’ve selected the guided workflow, we’ll match these users by email address. There will be no need to later complete the user import step for these users given they’ll have been brought into New Relic via your identity provider. You can continue the migration workflow and skip ahead to the **Migrate assets** step.
+
+### Login methods [#login-methods]
+
+If you chose **Manually** for [managing users](#managing-users), you have a choice for how those users log in. You can select either a) email/password login or b) single sign on (SSO). Note that SSO is available only for organizations with Pro or Enterprise edition.
```
Comment (mmfred) [inline src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:111]: available only for organizations with a Pro or Enterprise edition? The Pro or Enterprise edition?

**Review by mmfred — COMMENTED** [review]
Overall looks great!

File: src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx (hunk 12 above)
Comment (zuluecho9) [reply src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx:111]: i think 'with Pro or Enterprise edition' makes most sense to me but let me know if you think this seems clearly wrong
